lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -651,23 +762,38 @@ LogicalResult CppEmitter::emitAssignPrefix(Operation &op) {
   return success();
 }
 
+LogicalResult CppEmitter::emitLabel(Block &block) {
+  if (!hasBlockLabel(block)) {
+    return block.getParentOp()->emitError("Label for block not found.");
+  }
+  os << getOrCreateName(block) << ":\n";
+  return success();
+}
+
 static LogicalResult printOperation(CppEmitter &emitter, Operation &op) {
-  if (auto callOp = dyn_cast<mlir::CallOp>(op))
-    return printCallOp(emitter, callOp);
+  // emitc ops
   if (auto callOp = dyn_cast<emitc::CallOp>(op))
     return printCallOp(emitter, callOp);
+  if (auto constOp = dyn_cast<emitc::ConstOp>(op))
+    return printConstantOp(emitter, constOp);
   if (auto getAdressOfOp = dyn_cast<emitc::GetAddressOfOp>(op))
     return printGetAddressOfOp(emitter, getAdressOfOp);
   if (auto ifOp = dyn_cast<emitc::IfOp>(op))
     return printIfOp(emitter, ifOp);
-  if (auto yieldOp = dyn_cast<emitc::YieldOp>(op))
-    return printYieldOp(emitter, yieldOp);
   if (auto forOp = dyn_cast<emitc::ForOp>(op))
     return printForOp(emitter, forOp);
+  if (auto yieldOp = dyn_cast<emitc::YieldOp>(op))
+    return printYieldOp(emitter, yieldOp);
+  // std ops
+  // control flow
```

Comment:
```suggestion
  // Standard ops.
```

IMHO no need to add a comment on *control flow* if it's just a single op/printer function.